src/page/main-page/ui/video-block/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import React, { useEffect, useState } from 'react'
+import YouTube from 'react-youtube'
+
+import Wrap from '@/shared/ui/wrap'
+import Image from 'next/image'
+import img from '/public/main-page/video-block.jpg'
+import Icon from '@/shared/svg'
+
+import cl from './style.module.scss'
+import UseResizeVideo from '@/page/main-page/ui/video-block/lib/hooks'
+
+const VideoBlock = () => {
+  const [videoWatch, setVideoWatch] = useState<boolean>(false)
+  const [videoConfig, setVideoConfig] = useState({
+    height: '600',
+    width: '1065',
+    playerVars: {
+      autoplay: 1,
+    },
+  })
+
+  useEffect(() => {
+    if (videoWatch) {
+      document.body.style.overflow = 'hidden'
```

Comment:
We shouldn't refer to the document in react but I don't know if we can fix that in this case